### PR TITLE
Added hover effect on apps table items

### PR DESCRIPTION
**Pull request by sukhbir-singh**
now, used more standard colors for hover effect

hover_effect

### CODE REVIEW

File: public/themes/default/stylesheets/style.css [hunk 1]
```diff
@@ -32,6 +32,14 @@ a {
 #locale, #setting_dns select {
   border: 1px solid #cccccc; }
 
+.hover-style{
+	cursor: pointer;
+}
+
+.hover-style:hover{
+	background: ##d8d8d873;
```
Comment (cpg) [inline public/themes/default/stylesheets/style.css]: Is  ##d8d8d873 a valid color?

Comment (sukhbir-singh) [reply public/themes/default/stylesheets/style.css]: No it is not. Thanks for pointing it out. i pushed the changes.

**Comment by cpg** [comment]
While we're at this, how about if we also apply this to the other similar tables?

There are at least two in Users an Shares. There may be more (I don't recall).

**Comment by sukhbir-singh** [comment]
Okay sure! I will do it

**Comment by sukhbir-singh** [comment]
@cpg I have added this effect to other tables where-ever it is required in plugins. please review :smile:

**Pull request merged**